components/PageHeader/index.tsx

```diff
@@ -20,13 +20,19 @@ interface PageHeaderProps {
 }
 
 const PageHeader: React.FC<PageHeaderProps> = ({
-                                                 title,
-                                                 description,
-                                                 showBackButton = true,
-                                                 breadcrumbs,
-                                                 actions,
-                                                 className = ''
-                                               }: PageHeaderProps) => {
+  title,
+  description,
+  showBackButton = true,
+  breadcrumbs,
+  actions,
+  className = ''
+}: PageHeaderProps) => {
+
+  useEffect(() => {
+    document.title = `${title} | DMPTool`;
+    window.scrollTo(0, 0);
+  }, [title]);
```

Comment:
Added the setting of document.title and window.scrollTo() functionality that was previously in PageWrapper component